src/monitor-api-requests.spec.ts

```diff
@@ -388,3 +394,322 @@ describe("searchMonitor", () => {
+                  name: "regionName",
+                  available_values: [],
+                },
+                {
+                  defaults: ["{{aws_account.name}}"],
+                  prefix: "",
+                  name: "awsAccount",
+                  available_values: [],
+                },
+                {
+                  defaults: ["*"],
+                  prefix: "",
+                  name: "scope",
+                  available_values: [],
+                },
+              ],
+              tags: ["serverless_id:high_throttles", "created_by:dd_sls_app"],
+              integration: "serverless",
+              last_updated_at: 1696896000000,
+              meta_tags: ["product:serverless", "integration:amazon-lambda"],
+              name: "High Throttles on $functionName in $regionName for $awsAccount",
+              recommended_monitor_metadata: {
+                matcher: "serverless_id:high_throttles",
+              },
+              version: 2,
+              created_at: 1696896000000,
+            },
+          },
+        ],
+        meta: {
+          page: { total_filtered_count: 7, total_count: 7 },
+          facets: {
+            type: [{ count: 7, name: "query alert" }],
+            integration: [{ count: 7, name: "serverless" }],
+          },
+        },
+      }),
+    });
+    const response = await getRecommendedMonitors("datadoghq.com", "apiKey", "appKey");
+    expect(JSON.stringify(response)).toEqual(
+      JSON.stringify({
+        high_cold_start_rate: {
+          name: "High Cold Start Rate on $functionName in $regionName for $awsAccount",
+          threshold: 0.2,
+          message:
+            "More than 20% of the function’s invocations were cold starts in the selected time range. Datadog’s [enhanced metrics](https://docs.datadoghq.com/serverless/enhanced_lambda_metrics) and [distributed tracing](https://docs.datadoghq.com/serverless/distributed_tracing) can help you understand the impact of cold starts on your applications today. {{#is_alert}} Resolution: Cold starts occur when your serverless applications receive sudden increases in traffic, and can occur when the function was previously inactive or when it was receiving a relatively constant number of requests. Users may perceive cold starts as slow response times or lag. To get ahead of cold starts, consider enabling [provisioned concurrency](https://www.datadoghq.com/blog/monitor-aws-lambda-provisioned-concurrency/) on your impacted Lambda functions. Note that this could affect your AWS bill. {{/is_alert}}",
+          type: "query alert",
+          query: (cloudFormationStackId: string, criticalThreshold: number) => {
+            return `sum(last_15m):sum:aws.lambda.enhanced.invocations{cold_start:true,${cloudFormationStackId}} by {aws_account,functionname,region}.as_count() / sum:aws.lambda.enhanced.invocations{${cloudFormationStackId}} by {aws_account,functionname,region}.as_count() >= ${criticalThreshold}`;
```

Comment:
I'm wondering if this unit test caught [the typo](https://github.com/DataDog/serverless-plugin-datadog/pull/441/files#r1412523334) of `coudFormation` instead of `cloudFormation` or not.. Could you check and make sure it's working as intended?

Reply:
I think because cloudFormation is passed in as a param for a function, it returns [Function: query] instead of the actual query string. Here's what an example mocked response would look like: 

```
timeout: {
        name: 'Timeout on $functionName in $regionName for $awsAccount',
        threshold: 1,
        message: 'At least one invocation in the selected time range timed out. This occurs when your function runs for longer than the configured timeout or the global Lambda timeout. Resolution: [Distributed tracing](https://docs.datadoghq.com/serverless/distributed_tracing) can help you pinpoint slow requests to APIs and other microservices. You can also consider increasing the timeout of your function. Note that this could affect your AWS bill.',
        type: 'query alert',
        query: [Function: query]
      }
    }
```